### DISCUSSION TITLE
Suggestion for some features for a grocery WooCommerce store

### DISCUSSION THREAD

**Discussion by vvovkata** [inspireui/support#5202] [New Feature/Ideas Requests]
Hello, I would like to see a feature for decimal quantity input. Some products are sold per KG and clients can pick 0,75kg (or similar).  We send data from our NET APP to our WooCommerce website, which product is measurable or countable and put the measurable products in the specific product category. And all products which are in this category use decimal quantity input buttons.

Also, can you make these 2 plugins compatible: YITH points and rewards and YITH delivery date. They have constant updates and great support and a lot of people are using them. We have tried a lot and found out that YITH is best suited for our needs.

I would also like to ask if this payment gateway will work in the app? https://wordpress.org/plugins/woo-payment-gateway-paysera/
We use it on the website to accept card payments.

**Comment by nikolayng** [comment]
The YITH modules mentioned above and the the YITH affiliates are great. They all have better support, more regular updates and bigger user base. These plugins work best for our store. We don't use decimal quantity input for our products but I can see how this would be useful for some cases like flooring, construction materials or as vvovkata said grocery store. +1 for these features.

**Comment by HanielQC** [comment]
Dear @vvovkata , @nikolayng 

We are planning to develop FluxStore to support https://yithemes.com/themes/plugins/yith-woocommerce-points-and-rewards/?_1706790612
Could you please share your account by emailing hi@inspireui.com so our technical team can develop it? Thank you
